python/ray/util/client/worker.py

```diff
@@ -431,6 +438,62 @@ def ping_server(self) -> bool:
     def is_connected(self) -> bool:
         return self._conn_state == grpc.ChannelConnectivity.READY
 
+    def _server_init(self, job_config: JobConfig):
```

Comment:
```suggestion
    def _setup_runtime_env(self, job_config: JobConfig):
```

Reply:
Since it's calling ray.init, I feel _server_init is a better name?